apps/avatax/src/pages/api/webhooks/checkout-calculate-taxes.ts

```diff
@@ -152,6 +152,6 @@ const handler = checkoutCalculateTaxesSyncWebhook.createHandler(async (req, res,
  * TODO: Add tests to handler
  */
 export default wrapWithLoggerContext(
-  withOtel(withMetadataCache(handler), "/api/webhooks/checkout-calculate-taxes"),
+  withMetadataCache(wrapWithSpanAttributes(handler)),
```

Comment:
it'd be good if you add compose/pipe so it's easier to track whats going on

Reply:
I've plan to do this in the next PR